pkgs/development/python-modules/pyezminc/default.nix

```diff
@@ -0,0 +1,36 @@
+{ buildPythonPackage, isPy3k, fetchFromGitHub, stdenv,
+  netcdf, hdf5, libminc, ezminc,
+  cython, numpy, scipy
+}:
+
+buildPythonPackage rec {
+  pname = "pyezminc";
+  version = "1.2";
+  name = "${pname}-${version}";
+ 
+  disabled = isPy3k;  # ??
+  # sort of works but the example programs fail due to unicode issues, etc.
+
+  src = fetchFromGitHub {
+    owner  = "BIC-MNI";
+    repo   = "pyezminc";
+    rev    = "b9fc331c5ad296348fa1ac39df8b0e26ed667d85";
+    sha256 = "1qf115d82pyw956r9jcl9xxy03smfxz7ayj1chixd1yaxsqvcigs";
+  };
+
+  nativeBuildInputs = [ cython ];
+  buildInputs = [ netcdf hdf5 libminc ezminc ];
+  propagatedBuildInputs = [ numpy scipy ];
+
+  preBuild = "export CFLAGS='-fpermissive'";
+
+  doCheck = false;  # e.g., expects test data in /opt
+
+  meta = {
+    homepage = https://github.com/BIC-MNI/pyezminc;
+    description = "Python API for libminc using EZMINC";
+    license = stdenv.lib.licenses.gpl2;
+    maintainers = with stdenv.lib.maintainers; [ bcdarwin ];
+    platforms = stdenv.lib.platforms.unix;
```

Comment:
drop this line

Reply:
I don't understand this comment

Reply:
`platforms` is set automatically by `buildPythonPackage`

Reply:
removed